_publications/2013_PRDC_shrg.md

```diff
@@ -2,20 +2,20 @@
 layout: publication
 # The quotes make the : possible, otherwise you can do it without quotes
 title: "Towards Formal Approaches to System Resilience"
-key: 2013_PRDC_KULFI
+key: 2013_PRDC_shrg
 
 #YYYY_CONFSHORTNAME_PAPERSHORTNAME
 # paper | preprint | poster
 type: conference
-redirect_from: /publications/2013_PRDC_KULFI
+redirect_from: /publications/2013_PRDC_shrg
 
 # The shortname is used for auto-generated titels
-shortname: 2013_PRDC_KULFI
+shortname: 2013_PRDC_shrg
 # add a 2:1 aspect ratio (e.g., width: 400px, height: 200px) to the folder /assets/images/papers/
-image: 2013_PRDC_KULFI.png
+image: 2013_PRDC_shrg.png
 # 2018_tvcg_lineage.png
 # add a 2:1 aspect ratio teaser figure (e.g., width: 1200px, height: 600px) to the folder /assets/images/papers/
-image_large: 2013_PRDC_KULFI.png
+image_large: 2013_PRDC_shrg.png
```

Comment:
You should update the rest of the file as well. Talk to someone (Mark, Shaobo, Maryam?) to learn how to do that.